### DISCUSSION TITLE
issues while integrating Admob and firebase on iOS

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#2184] [Q&A]
I tried to integrate Admob & firebase on iOS with pod and I got stuck on this:
```
ld: framework 'Pods_LudoX' not found
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

LudoX is my project name and Pods_LudoX is already listed there 'Target Support Files' in project navigator. 
And Pods_LudoX.framework is also there in 'Frameworks, Libraries, and Embedded Content'

my pod looks like this: 
```
target 'LudoX' do
  # Comment the next line if you don't want to use dynamic frameworks
  #inherit! :search_paths
  use_frameworks!

  # Pods for LudoX
    pod 'GoogleUserMessagingPlatform'
    pod 'Google-Mobile-Ads-SDK'
    pod 'FirebasePerformance'
    pod 'FirebaseAnalytics'
    pod 'FirebaseCrashlytics'
    pod 'FirebaseRemoteConfig'
end
```

Do I need to make any changes in CmakeLists.txt ?

**Comment by rh101** [comment]
Is there any reason the line `inherit! :search_paths` is commented out?

Also, the comment `# Comment the next line if you don't want to use dynamic frameworks` is for `use_frameworks!`, not for `inherit! :search_paths`, in case that is the reason you have it commented out.

Where is your prodfile located within the project?

Also, there are other posts on this discussion board regarding podfiles, so do a quick search and you should find them.

**Reply by meehjeevan** [reply]
podfile is in build_ios_arm64 folder where the .xcodeproj is created.
with or without commented out `inherit! :search_paths` it's not making any difference. I'm still getting the same errors.

**Reply by rh101** [reply]
Right, well take a look at this thread for anything that may help: https://github.com/axmolengine/axmol/discussions/1817

**Reply by arungowda09** [reply]
Issue resolved ?
@meehjeevan
